game/src/main/java/net/demilich/metastone/game/entities/weapons/Weapon.java

```diff
@@ -94,4 +89,9 @@ public String toString() {
 		result += "]";
 		return result;
 	}
+
+	public void setDurability(int durability) {
```

Comment:
Where would we use setDurability(int)?  For now it a no-op, do you want to commit it like this?